mock/py/mockbuild/util.py

```diff
@@ -1366,23 +1366,25 @@ def check_config(config_opts):
 
 
 @traceLog()
-def include(config_file, config_opts, is_statement=False):
+def include(config_file, config_opts):
     if not os.path.isabs(config_file):
         config_file = os.path.join(config_opts['config_path'], config_file)
 
+    str_mx = re.compile(r'^\s*include\((.*)\)')
     if os.path.exists(config_file):
-        if is_statement and config_file in config_opts['config_paths']:
+        if config_file in config_opts['config_paths']:
             getLog().warning("Multiple inclusion of %s, skipping" % config_file)
-            return
+            return ""
 
         config_opts['config_paths'].append(config_file)
-
         with open(config_file) as f:
             content = f.read()
-            content = re.sub(r'include\((.*)\)', r'include(\g<1>, config_opts, True)', content)
-            code = compile(content, config_file, 'exec')
-        # pylint: disable=exec-used
-        exec(code)
+            res = str_mx.search(content)
+            if res is not None:
+                sub_config_file = eval(res.groups()[0])
```

Comment:
Why the eval() here?

Reply:
eval returns the value of string under include parentheses

Reply:
> What if I have two includes on one line?

I forgot  this case , ATM it won't work we need one include per line .

Reply:
Ah, now I see what you are trying to achieve.

Reply:
> eval returns the value of string under include parentheses

What if we parsed the `include` statement properly, accepting only valid pathname characters wrapped by quotes and double quotes?